TestProjects/VisualEffectGraph_HDRP/Assets/AllTests/Editor/Tests/VFXCustomSpawnerRecordSpawnCount.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using UnityEngine;
+using UnityEngine.VFX;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace UnityEditor.VFX.Test
+{
+    class VFXCustomSpawnerRecordSpawnCount : VFXSpawnerCallbacks
+    {
+        public class InputProperties
+        {
+        }
+
+        public static List<int> s_ReceivedSpawnCount = new List<int>();
```

Comment:
It seems to be used only in unit tests, but I feel that leaving this public static field writable is dangerous.

Reply:
Fix (kind of) at f6f92e9ea7791284fc7bcf33fba078dfb52c58f1
I think I could create a common analyze helper for all these custom test spawner (using local delagate)